gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
-distributionBase=PROJECT
+distributionBase=GRADLE_USER_HOME
```

Comment:
commitujemy tylko pliki .java i zmiany w tych plikach - już zwracałem Ci na to uwagę i 3 raz nie będę pobłażliwy 😉